setup.py

```diff
@@ -618,12 +617,13 @@ def sundry_functions(cls) -> str:
 
 
 def retrieve_blobs_sidecar(slot: Slot, beacon_block_root: Root) -> BlobsSidecar:
-    pass'''
+    return "TEST"'''
```

Comment:
Fixed in af48987c2

Reply:
Hmm, slightly annoying because `None` could also mean that the sidecar is not available (I know in python we would do this by raising an exception but not all languages would have that available).